language/mixtral-8x7b/user.conf

```diff
@@ -2,4 +2,5 @@
 # The key has the format 'model.scenario.key'. Value is mostly int64_t.
 # Model maybe '*' as wildcard. In that case the value applies to all models.
 # All times are in milli seconds
-#
\ No newline at end of file
+#
+mixtral-8x7b.*.use_token_latencies = 1
```

Comment:
Is this needed if it's already in `mlperf.conf`?
https://github.com/mlcommons/inference/blob/master/mlperf.conf#L62

Reply:
It's not necessary (but good to have). mlperf.conf will take precedence over this.

Reply:
For issue #1787, the FromConfig function will ignore both values if this is not passed.